.golangci.yml

```diff
@@ -15,6 +15,10 @@ linters:
     - interfacer
     - lll
     - scopelint
+    - wsl # added in 1.20
+    - gocognit # added in 1.20
+    - whitespace # added in 1.19
```

Comment:
Would be nice to keep this list sorted alphabetically so that it's easier to search.